specs/README.md

```diff
@@ -0,0 +1,9 @@
+# Specs
+
+Here you'll find specifications for fast-dna custom elements and other library features.
```

Comment:
Generally, we have always used  FAST-DNA in all caps.  I prefer the reading in lower case, but, as a name we should maintain casing.  I actually think we could just remove fast-dna all together.  "... find specifications for custom"